src/util-logopenfile.c

```diff
@@ -256,6 +256,31 @@ SCConfLogOpenGeneric(ConfNode *conf,
     if (append == NULL)
         append = DEFAULT_LOG_MODE_APPEND;
 
+    /* JSON flags */
+    ConfNode *json_flags = ConfNodeLookupChild(conf, "json");
```

Comment:
It looks like ConfNodeLookupChildValue just returns NULL if the node is NULL. It might be sensible to check it anyway and skip doing the lookups if it json_flags is NULL. What do you think?